Formula/easy-rsa.rb

```diff
@@ -1,18 +1,14 @@
 class EasyRsa < Formula
   desc "Homebrew formula for `easy-rsa`"
-  homepage 'https://github.com/OpenVPN/easy-rsa'
-  url "https://github.com/OpenVPN/easy-rsa/archive/v3.0.5.tar.gz"
-  sha256 "01d0a0b0105fb80742455cef8b6336fcb67bf69c8675d3bbd66fdf741fd6d34f"
+  homepage "https://github.com/OpenVPN/easy-rsa"
+  url "https://github.com/OpenVPN/easy-rsa/archive/v3.0.6.tar.gz"
+  sha256 "2a87c0c617bbfab556c541b83fa5bfdf1c244bfb16b530ca12efb3d979e85191"
   head "https://github.com/OpenVPN/easy-rsa.git"
 
-  devel do
-    version '3.0.4'
-  end
-
-  def install    
+  def install
     inreplace "easyrsa3/vars.example",
               "#set_var EASYRSA\t\"${0%/*}\"",
-              "#set_var EASYRSA \"#{Formula["easy-rsa"].opt_prefix}/share\""    
+              "#set_var EASYRSA \"#{Formula["easy-rsa"].opt_prefix}/share\""
```

Comment:
I'm not sure, but I feel like this might be an example of the misconfiguration described in [hound Issue #977](https://github.com/houndci/hound/issues/977)?